extensions/schema-compare/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "schema-compare",
+  "displayName": "SQL Server Schema Compare",
+  "description": "SQL Server Schema Compare for Azure Data Studio supports comparing the schemas of databases and dacpacs.",
+  "version": "0.1.0",
+  "publisher": "Microsoft",
+  "preview": true,
+  "engines": {
+    "vscode": "^1.25.0",
+    "sqlops": "*"
+  },
+  "license": "https://raw.githubusercontent.com/Microsoft/azuredatastudio/master/extensions/import/Microsoft_SQL_Server_Import_Extension_and_Tools_Import_Flat_File_Preview.docx",
+  "icon": "images/sqlserver.png",
+  "aiKey": "AIF-5574968e-856d-40d2-af67-c89a14e76412",
+  "activationEvents": [
+    "*"
+  ],
+  "main": "./out/main",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Microsoft/azuredatastudio.git"
+  },
+  "extensionDependencies": [
+    "Microsoft.mssql"
+  ],
+  "contributes": {
+    "commands": [
+      {
+        "command": "schemaCompare.start",
+        "title": "Schema Compare",
+        "icon": {
+          "light": "./images/light_icon.svg",
+          "dark": "./images/dark_icon.svg"
+        }
+      }
+    ],
+    "menus": {
+      "objectExplorer/item/context": [
+        {
+          "command": "schemaCompare.start",
+          "when": "connectionProvider == MSSQL && nodeType && nodeType == Database",
+          "group": "export"
+        }
+      ]
+    }
+  },
+  "dependencies": {
+    "vscode-extension-telemetry": "0.0.18",
+    "vscode-nls": "^3.2.1"
```

Comment:
I don't see any telemetry calls in here - were you going to add any? It's always a good idea to get some basic stats like launches and command executions

Reply:
I'll add telemetry calls in a later PR